src/main/java/org.wso2.apim.monetization/impl/StripeMonetizationConstants.java

```diff
@@ -119,10 +119,11 @@
     public static final String ITEMS = "items";
     public static final String ACTION = "action";
     public static final String INCREMENT = "increment";
-    public static final String TIME_FORMAT = "yyyy-MM-dd HH:mm:ss";
+    public static final String TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss";
+    public static final String TIMEZONE_FORMAT = ".023+05:30";
     public static final String QUANTITY = "quantity";
     public static final String TIMESTAMP = "timestamp";
-    public static final String TIME_ZONE = "UTC";
+    public static final String TIME_ZONE = "IST";
```

Comment:
is this the default timezone? Isn't it better to provide a config for this?

Reply:
For now only UTC will be usaed and that property is not used.  The property is changed back in 09f0f2a0ec6b148f13a8092bf53055c9df346ef2